### ISSUE TITLE
[Bug] Bottom of text cut off in search suggestions

### ISSUE THREAD

**Issue by eliserichards** [duckduckgo/Android#4564]
### Describe the bug

The bottom of the suggested text when searching is slightly cut off:
<img src="https://github.com/duckduckgo/Android/assets/43795363/79ab91b6-36cb-48b2-91b5-5e447cb9c284" width="300">

Reproducible on both `develop` and `main`.

### How to Reproduce

1. Download the app (or open current version) 
2. Click into the URL bar
3. Start typing a search

### Expected behavior

Suggested text is shown fully, not cut off

### Environment

```markdown
- DDG App Version: v5.200.1 (52001000)
- Device: Pixel 3
- OS: Android 12
```

**Comment by VISHNUONE8** [comment]
@eliserichards the issue has been solved in the latest version of ddg 5.201.1 
please check and confirm if the issue still persists for you.

**Comment by joshliebe** [comment]
@eliserichards This has been fixed with https://github.com/duckduckgo/Android/pull/4588 and will roll out with the next release.
